db/seeds.d/106-mail_notifications.rb

```diff
@@ -30,7 +30,7 @@
 ]
 
 notifications.each do |notification|
-  ::MailNotification.find_or_create_by_name(notification)
+  ::MailNotification.create(notifications) unless ::MailNotification.where(:name => notification[:name]).exists?
```

Comment:
@johnpmitsch typo there in the statement, It should ve been ```::MailNotification.create!(notification)``` instead of ```::MailNotification.create(notification)```. @daviddavis 's looks more railsey . Lets go with that .